.github/drafts/16521/16521-Code-Scanner-Outline.md

```diff
@@ -0,0 +1,73 @@
+|-----------------|----------------------|---------------------------------------------------------------------------------------------------------------------|-----------------------------|
+| Text            | Formatting           | Sets the label text of the widget.                                                                                  | -                           |
+| Visible         | Binding & Formatting | Controls widget's visibility on the page. When turned off, the widget will not be visible when the app is published | {{CodeScanner1.isVisible}}  |
+| Disabled        | Binding & Formatting | Disables input to the widget. The widget will remain visible to the user but a user input will not be allowed.      | {{CodeScanner1.isDisabled}} |
+| Animate Loading | Formatting           | Controls the loading of the widget.                                                                                 | -                           |
+| Tooltip         | Formatting           | It sets a tooltip for the widget. You can add hints or extra information about the required input from the user.    | -                           |
+| value           | Binding              | Fetches the value of scanned code                                                                                   | {{CodeScanner1.value}}      |
+|                 |                      |                                                                                                                     |                             |
+
+
+**//cross-link common properties** 
+
+### Text
+
+### Visible
+
+### Disabled
+
+ ### Animate Loading
+
+### Tooltip
+
+### Value
+
+
+# Events
+
+Event Intro
+| Event          |  Description                                     | Example                       | Code Snippet                                                      |
+|----------------|--------------------------------------------------|-------------------------------|-------------------------------------------------------------------|
+| onCodeDetected | Triggers an action when a valid code is detected | <Describe the example usage.> | <Add code snippets for reading data when the event is triggered.> |
+
+### onCodeDetected 
+
+
+# Styles
+  Style properties allow you to change the look and feel of the widget. 
+  | Style          | Description                                            |
+|----------------|--------------------------------------------------------|
+| Icon           | Sets an icon to be included in the input field.        |
+| Position       | Sets the label position of the widget.                 |
+| Placement      | Sets the label alignment of the widget.                |
+| Button Color   | Allows you to set color for the button.                |
+|  Border Radius | Allows you to define curved corners.                   |
+| Box Shadow     | Allows you to choose from the available shadow styles. |
+
+## Conclusion
+## Cross Reference Section
+
```

Comment:
Could you please add the reference links that you think will be added here? Also, please add a cross-reference to troubleshooting section.

Reply:
Made necessary changes.